charts/trino/values.yaml

```diff
@@ -447,6 +447,15 @@ secretMounts: []
 # ```
 
 coordinator:
+  deployment:
+    progressDeadlineSeconds: ~
```

Comment:
This needs a comment explaining what it does and why users would want to configure it. You can get that from Kubernetes docs and even reference them if there's some longer explanation and/or good examples. Also see how we do this for other properties